tls/s2n_resume.h

```diff
@@ -36,7 +36,11 @@
 #define S2N_TICKET_AAD_LEN              (S2N_TICKET_AAD_IMPLICIT_LEN + S2N_TICKET_KEY_NAME_LEN)
 #define S2N_AES256_KEY_LEN              32
 #define ONE_SEC_IN_NANOS                1000000000
-#define S2N_TICKET_SIZE_IN_BYTES        (S2N_TICKET_KEY_NAME_LEN + S2N_TLS_GCM_IV_LEN + S2N_STATE_SIZE_IN_BYTES + S2N_TLS_GCM_TAG_LEN)
+#define ONE_WEEK_IN_SECS                604800
+#define S2N_TICKET_SIZE_IN_BYTES        (S2N_TICKET_KEY_NAME_LEN + S2N_TLS_GCM_IV_LEN +     \
```

Comment:
Do we still need S2N_TICKET_SIZE_IN_BYTES? If so, should we rename it S2N_TLS12_TICKET_SIZE_IN_BYTES?